src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -111,6 +124,7 @@
 
 class GitHubEnterpriseIntegration(IntegrationInstallation, GitHubIssueBasic, RepositoryMixin):
```

Comment:
`GitHubIntegration` extends from `CommitContextMixin` too. Github Enterprise should have commit metadata too.

Reply:
This is true, also looking at some of the procedure for CODEOWNERS, it looks like `GitHubEnterpriseIntegration` will also need the `codeowners_locations` set for it to work.

Likely also makes sense to set `repo_search` to `True`

Reply:
minor correction to my above comment* The `get_commit_context` and `CommitContextMixin` are for the suspect commits feature, not stack trace linking.